<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemTypeView</class>
 <widget class="QWidget" name="ItemTypeView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="itemTypeBox"/>
   </item>
   <item>
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="descEdit"/>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Dodaj</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Odswiez</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>ItemTypeView</receiver>
   <slot>addType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ItemTypeView</receiver>
   <slot>updateType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addType()</slot>
  <slot>updateType()</slot>
 </slots>
</ui>
